### PR TITLE
Application will not send if user is on application page and deadline passes

### PR DESCRIPTION
**Pull request by IanWearsHat**
- `handleSubmit` in `Form.tsx` will reload the page if the deadlines has passed and NOT submit the form
- "If you already applied" message in `ApplicationsClosed.tsx` will not show if user is logged in, but hasn't submitted an application

### CODE REVIEW

**Comment by IanWearsHat** [comment]
Note: because there is no way to access the `/apply` route from the ui, the Applications Closed message really can't be viewed unless the user manually adds `/apply` to the url.

File: apps/site/src/app/(main)/apply/sections/ApplicationsClosed/ApplicationsClosed.tsx [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
-			<hr className="mt-5 w-full h-0.5 bg-[#432810]" />
-			<p className="pt-10 text-lg">
-				If you already applied, please{" "}
-				<Link href="/login" className="text-blue-600 underline">
-					log in
-				</Link>{" "}
-				to visit your applicant portal.
-			</p>
+			<hr className="my-10 w-full h-0.5 bg-[#432810]" />
+			{(identity.uid === null || identity.status !== null) && (
```
Comment (taesungh) [inline apps/site/src/app/(main)/apply/sections/ApplicationsClosed/ApplicationsClosed.tsx:14]: If a user has already applied, they would be redirected to Portal anyway, so I don't think the status check is necessary.

File: apps/site/src/app/(main)/apply/sections/Form/Form.tsx [hunk 2]
```diff
@@ -25,6 +26,12 @@ export default function Form() {
 	): Promise<void> => {
 		// Disable native post submission
 		event.preventDefault();
+
+		if (hasDeadlinePassed()) {
+			window.location.reload();
```
Comment (taesungh) [inline apps/site/src/app/(main)/apply/sections/Form/Form.tsx:31]: Chore: Could you include a comment to indicate this is to show the messaged that applications have closed.

**Review by taesungh — APPROVED** [review]
Thanks for adding, looks alright. Hopefully not too scary for people trying to submit last minute.

**Review by samderanova — APPROVED** [review]

**Pull request merged**